frontend/src/container/LogDetailedView/TableView.tsx

```diff
@@ -284,19 +290,44 @@ function TableView({
 
 				const fieldFilterKey = filterKeyForField(fieldData.field);
 
+				const bodyHtml =
+					record.field === 'body'
+						? {
+								__html: convert.toHtml(
+									dompurify.sanitize(unescapeString(record.value), {
+										FORBID_TAGS: [...FORBID_DOM_PURIFY_TAGS],
+									}),
+								),
+						  }
+						: { __html: '' };
+
 				return (
 					<div className="value-field">
-						<CopyClipboardHOC textToCopy={textToCopy}>
-							<span
-								style={{
-									color: Color.BG_SIENNA_400,
-									whiteSpace: 'pre-wrap',
-									tabSize: 4,
-								}}
-							>
-								{removeEscapeCharacters(fieldData.value)}
-							</span>
-						</CopyClipboardHOC>
+						{record.field === 'body' ? (
+							<CopyClipboardHOC textToCopy={textToCopy}>
+								<span
+									style={{
+										color: Color.BG_SIENNA_400,
+										whiteSpace: 'pre-wrap',
+										tabSize: 4,
+									}}
+									// eslint-disable-next-line react/no-danger
+									dangerouslySetInnerHTML={bodyHtml}
```

Comment:
Check for any security issue.

Reply:
i remember the conversation, we are using dompurify everywhere so no such risk with this. already using this in logs raw and other views